src/test/java/core/basesyntax/model/UserTest.java

```diff
@@ -0,0 +1,150 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
```

Comment:
There is no need to test model classes
It is necessary to write unit tests for services
integration for controllers and repositories

Reply:
Got it! Thank you ❤️